homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language that allows us to add interative to our website by implement complex features on web pages. 
```

Comment:
On the contrary JavaScript is more than just a programming language for web interactivity and answering the question the way you did, is for another question let me explain myself.

So JavaScript ever since 2009 when NodeJS was introduced one could now build web services and CLI's with JavaScript, come 2015 and React Native was introduced and you could now create Native Mobile Applications with JavaScript and finally 2019 came ElectronJS one can now build Desktop Applications with JavaScript...
So my point here is JavaScript is more than what you say it is and adding "Web interactivity" you basically answering "what JavaScript does" not "what JavaScript is".

### The key here to this question is to distinguish "what JavaScript is" and not "what JavaScript does" cause that is another topic and broad for that matter